Sources/Sentry/SentryFramesTracker.m

```diff
@@ -158,11 +168,32 @@ - (void)displayLinkCallback
     self.previousFrameTimestamp = lastFrameTimestamp;
 }
 
+- (void)sendTimestampSpanWithStart:(CFTimeInterval)start
+                               end:(CFTimeInterval)end
+                          isFrozen:(BOOL)isFrozen
+{
+    __block id<SentrySpan> frameSpan;
+    [SentrySDK.currentHub.scope useSpan:^(id<SentrySpan> _Nullable span) {
+        frameSpan = [span startChildWithOperation:@"frame.info"
```

Comment:
Not sure what a good operation name is here.

Reply:
Im checking this with team.